### PR TITLE
style(vars): FRMW-594 Update Link Global Vars

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-594

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

### CODE REVIEW

File: src/styles/vars.less [hunk 1]
```diff
@@ -427,6 +427,8 @@
 
 // # Text Colors
 // deprecated
+@link-color: @app-link-text-color;
+@link-hover-color: saturate(@app-link-hover-text-color, 15%);
```
Comment (glynnis) [inline src/styles/vars.less]: @ssalinas24 Your definitions here use the newer vars, `@app-link-hover-text-color` and `@app-link-text-color`. Instead, please preserve the original calculation/var and comment to indicate what the new var is called, as we've done elsewhere (see https://github.com/rackerlabs/encore-ui/blob/master/src/styles/vars.less#L472).

Comment (glynnis) [inline src/styles/vars.less]: Ah, nevermind -- I looked at the JIRA story and see what you did here :) Deleting this and my prior comment to give an LGTM

**Comment by glynnis** [comment]
Design LGTM

File: src/styles/vars.less (hunk 1 above)
Comment (ssalinas24) [inline src/styles/vars.less]: @glynnis actually I think your instinct was correct. I actually thought i did what you were trying to suggest, but i didn't. I should probably be doing `@link-hover-color: saturate(@link-color, 15%);`

Comment (glynnis) [inline src/styles/vars.less]: Hah, ok. I saw in the story on JIRA that you'd specified how to deprecate so I thought there was a good reason for that (it being a more recent variable that you were deprecating). So confusing.

File: src/styles/vars.less [hunk 2]
```diff
@@ -427,6 +427,8 @@
 
 // # Text Colors
 // deprecated
+@link-color: @app-link-text-color;
+@link-hover-color: saturate(@link-color, 15%);
```
Comment (ssalinas24) [inline src/styles/vars.less]: @glynnis here is the corrected line.

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
@ssalinas24, please rebase

**Comment by ssalinas24** [comment]
@CITguy alright I rebased and pushed back up again.

**Pull request merged**